uptane-standard.md

```diff
@@ -677,6 +677,7 @@ Note that one of the ECU version reports should be the version report for the pr
 
 An ECU version report is a metadata structure that MUST contain the following information:
 
+* A token (nonce) for the time server. The primary ECU will eventually send this token to the time server, which will sign the token and send it back.
```

Comment:
```suggestion
* A token (nonce) for a time server, if any is used. The primary ECU will eventually send this token to the time server, which will sign the token as well as the current time, and send them back together.
```

Reply:
Is the time server meant to be optional? If so, we should revise [section 5.3](https://uptane.github.io/uptane-standard/uptane-standard.html#server-repository-implementation-requirements), because right now it's a SHALL.

Reply:
I interpreted @trishankatdatadog's comment to mean the nonce was optional, not the time server.

Reply:
Yes, the time server is meant to be optional. However, if it is used, the nonce is then not optional. We should make this clearer...

Maybe @mnm678 can help?